recipes/matio/all/test_package/test_package.c

```diff
@@ -0,0 +1,11 @@
+#include "matio.h"
```

Comment:
Not sure why it was changed from bracket inclusion `matio.h` to quoteed inclusion `"matio.h"`. This is not consistent among the available recipes in the repository.